scripts/data_contamination/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+RUN mkdir -p /script/src
+
+WORKDIR /script/src
+
+COPY . .
+
+RUN pip install -r ./requirements-freeze.txt
+
+ENTRYPOINT python compute_contamination_metrics.py 
```

Comment:
You have to use [exec form](https://docs.docker.com/engine/reference/builder/#entrypoint) in order for arguments to be appended to `ENTRYPOINT`. Concretely, this looks like:

```
ENTRYPOINT ["python", "compute_contamination_metrics.py"]
```

Reply:
Ah I see, thanks for the pointer, updated